lib/spack/spack/test/web.py

```diff
@@ -407,9 +408,9 @@ def mock_verify_locations(self, cafile, capath, cadata):
         """overwrite ssl's verification to simply check for valid file/path"""
         assert cafile or capath
         if cafile:
-            assert os.path.isfile(cafile)
+            assert Path(cafile).is_file()
```

Comment:
Rather than casting on usage, cast at the beginning of the method so you can just use the pathlib api without casting everywhere.